storage/bootstrap/bootstrapper/peers/source_test.go

```diff
@@ -166,7 +166,6 @@ func TestPeersSourceReturnsFulfilledAndUnfulfilled(t *testing.T) {
 
 func TestPeersSourceIncrementalRun(t *testing.T) {
 	for _, cachePolicy := range []series.CachePolicy{
-		series.CacheAll,
```

Comment:
is this an intentional change?

Reply:
nah was debugging something else and forgot to put it back in, fixed. Thanks